source/1_engine/rex_engine/rex_engine.sharpmake.cs

```diff
@@ -25,9 +26,11 @@ public RexEngine() : base()
     string relative_source_path = Util.PathGetRelative(Path.Combine(Globals.SourceRoot), SourceRootPath);
     GenerationConfigPath = Path.Combine(Globals.Root, "config", relative_source_path, "generation.json");
     MemoryTagsHeaderFile = Path.Combine(SourceRootPath, "include", "rex_engine", "memory", "memory_tags.h");
+    FileSystemMountHeaderFile = Path.Combine(SourceRootPath, "include", "rex_engine", "filesystem", "mount_roots.h");
```

Comment:
Why do we need this?

Reply:
to avoid typos when deciding where memory goes.
this might seem pointless in a small framework like ours but useful when it grows bigger.
also makes it better to maintain if we ever decide to put data elsewhere (eg. test vs non-test)